docs/protocol/substrate-cli.mdx

```diff
@@ -147,9 +147,9 @@ We will be downloading two files for your respective operating system.
 
   3. Create the `subspace` directory and the `docker-compose.yaml` file in it. Paste the output from the generator into the file.
   
-  4. Now go to directory with `docker-compose.yaml` and type `docker-compose up -d` to start everything
+  4. Now go to directory with `docker-compose.yaml` and type `docker compose up -d` to start everything
```

Comment:
Please also update the versioned version of the same file, which can be found in `/versioned-docs/version-latest/protocol/substrate-cli.mdx`

Reply:
Thanks, guys for adding this issue and PR on my behalf. (https://discord.com/channels/864285291518361610/1143624977216712825/1144008595323756685)

Topper